README.md

```diff
@@ -523,7 +513,7 @@ INFO [2024-04-18 10:48:47.231]   chain simulator's is accessible through the URL
 
 ### Build docker image
 ```
-make docker-build
+DOCKER_BUILDKIT=1 docker build -t chainsimulator:latest .
```

Comment:
Is this ok?

Reply:
Rolled this back to what's in master branch